daemon/daemon.go

```diff
@@ -591,6 +591,11 @@ func (daemon *Daemon) restore(cfg *configStore) error {
 	for _, c := range containers {
```

Comment:
The comment above now only applies to the `if` branch ... could move it into there?